module/PowerNSX.psm1

```diff
@@ -4941,12 +4941,12 @@ function Get-NsxClusterStatus {
     .EXAMPLE
     This example shows how to query the status for the cluster MyCluster
 
-    PS C:\> get-cluster MyCluster | Get-NsxClusterStatus
+    get-cluster MyCluster | Get-NsxClusterStatus
 
     .EXAMPLE
     This example shows how to query the status for all clusters
 
-    PS C:\> get-cluster MyCluster | Get-NsxClusterStatus
+    get-cluster MyCluster | Get-NsxClusterStatus
```

Comment:
remove the cluster MyCluster from this one

Reply:
I've not gone through the examples to normalise yet other than the find and replace on PS C:\>

I'll add this to sweep No. 2